horovod/common/operations.cc

```diff
@@ -2245,18 +2714,24 @@ int horovod_size() {
 }
 
 int horovod_local_size() {
+#if HAVE_MLSL
+  return horovod_size();
```

Comment:
`local_size` = a number of processes that run on the same node or container / can use shared memory to communicate.  What is the way you typically run Horovod with MLSL, is it a single process per node or multiple processes per node?